### PR TITLE
add test for #3277

### PR DESCRIPTION
**Pull request by hone**

### CODE REVIEW

**Comment by indirect** [comment]
I don't think this is the underlying cause of #3277

**Comment by indirect** [comment]
Er, I mean, I don't think it is only sudo installs that cause Bundler to leave empty directories behind in /tmp. Especially since this test passes, and Bundler (apparently) still leaves empty directories behind in /tmp. :)

**Comment by segiddins** [comment]
@indirect we're pretty sure that it only did that pre-1.8

**Comment by indirect** [comment]
oh. hmmmm....

**Comment by indirect** [comment]
Nope, that test still passes on 1-7-stable. :/

**Comment by hone** [comment]
@indirect hmm. removing the cleanup stuff made the test fail on my computer locally.

File: spec/install/gems/sudo_spec.rb [hunk 1]
```diff
@@ -106,6 +109,20 @@
       expect(default_bundle_path("gems/rack-1.0.0")).to exist
       should_be_installed "rack 1.0"
     end
+
+    it "cleans up the tmpdirs generated", focused: true do
```
Comment (segiddins) [inline spec/install/gems/sudo_spec.rb]: 1.9 hash syntax is no good

Comment (hone) [inline spec/install/gems/sudo_spec.rb]: focused tests are also bad. let me fix that.

**Pull request merged**